satellite/analytics/service.go

```diff
@@ -77,6 +90,20 @@ const (
 	Personal UserType = "Personal"
 )
 
+// PayloadUser is to create user in HubSpot.
+type PayloadUser struct {
```

Comment:
nit `PayloadUser` and `PropertiesUser` are only used internally in this package, so it would be okay to make them private naming them `payloadUser`, `propertiesUser`